hosting/static-website/README.md

```diff
@@ -0,0 +1,128 @@
+# Static website
+The example shows how to deploy a simple, static website on the IC in an asset canister. The website is very simple, it just shows the Dfinity logo and has a basic styling of the logo. 
+
+![Website](README_images/website.png)
+
+## Introduction
+The purpose of this example is to show how to deploy a static website in an asset canister. While the website in this example is very simple, the method is the same for more advanced static website, e.g. based on popular static site generators.
+
+This example covers:
+
+- Build a very simple HTML website
+- Create the dfx.json file
+- Deploy the canister smart contract locally
+- Test the frontend in browser 
+
+## Installation
+This example project can be cloned, installed and deployed locally, for learning and testing purposes. The instructions are based on running the example on either macOS or Linux, but when using WSL2 on Windows, the instructions will be the same.
+
+### Prerequisites
+The example project requeres the following installed:
+
+- GitHub
+- DFX 
+
+GitHub can be installed from various package managers. DFX can be installed following the instructions [here](https://smartcontracts.org/docs/quickstart/local-quickstart.html#download-and-install).
+
+### Install
+Install the example dapp project:
+
+```bash
+$ git clone https://github.com/dfinity/examples
+$ cd motoko/static-website
+```
+
+## Documentation
+There are two parts in this dapp, one is the website and the other is the `dfx.json`. Since there's no backend, there is not any benefit of using the `dfx new project_name` to setup a template, the `dfx.json` file is all there is needed.
+
+### Website
+The website is really simple. It consists of an HTML file, a CSS file and a PNG file. The content of the HTML file looks like this:
+
+```html
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width">
+        <title>Static Website</title>
+        <base href="/">
+        <link type="text/css" rel="stylesheet" href="styles.css" />
+    </head>
+    <body>
+        <img src="logo.png" alt="DFINITY logo" />
+    </body>
+</html>
+```
+
+The only styling done in the CSS file is aligning the logo image:
+
+```css
+img {
+    max-width: 50vw;
+    max-height: 25vw;
+    display: block;
+    margin: auto;
+}
+```
+
+The top level folder is called `assets` and inside this folder, the CSS file and image is stored in a folder called `assets` and the HTML file `index.html` is stored in a folder called `src`. 
```

Comment:
For the purposes of this simple example, since these three files end up deploying into the same place, would it make sense to move them into a single directory?  This would avoid having an `assets/assets/` directory.
```
hosting/
    static-website/
        assets/
            index.html
            main.css
            logo.png
```